new-log-viewer/src/services/LogFileManager/index.ts

```diff
@@ -27,39 +30,49 @@ import {
 } from "./utils";
 
 
+const SEARCH_CHUNK_SIZE = 10000;
+
 /**
  * Class to manage the retrieval and decoding of a given log file.
  */
 class LogFileManager {
+    readonly #fileName: string;
+
+    readonly #numEvents: number = 0;
+
     readonly #pageSize: number;
 
-    readonly #fileName: string;
+    #queryId: number = 0;
```

Comment:
Depending on the discussion at https://github.com/y-scope/yscope-log-viewer/pull/80/files#r1802086908 , shall we rename this as `nextQueryId` (if you agree with the proposal in the other discussion) or `lastQueryId?